moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -158,9 +158,10 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
   std::vector<double> joint_max_vector_;
   std::vector<bool> joint_has_limits_vector_;
   std::vector<std::string> link_names_;
-  size_t num_joints_;
+  const size_t num_joints_;
   std::vector<int> free_params_;
   bool active_;  // Internal variable that indicates whether solvers are configured and ready
+  const std::string name_{ "ikfast" };
```

Comment:
Exotic syntax. Why not just use `const std::string name_ = "ikfast"`?

Reply:
"C++11 attempts to overcome the problems of C++03 initialization by introducing a universal initialization notation that applies to every type—whether a POD variable, a class object with a user-defined constructor, a POD array, a dynamically allocated array, or even a Standard Library container. The universal initializer is called a brace-init."

http://www.informit.com/articles/article.aspx?p=1852519

Although non-standard in this code written previous to C++11, I'm fine with it

Reply:
nice!